dnf-docker-test/features/module-info-1.feature

```diff
@@ -1,14 +1,157 @@
+           Artifacts +: +TestA-1-2.modA.noarch
+            +: +TestB-1-1.modA.noarch
+            +: +TestC-1-2.modA.noarch
+            +: +TestD-1-1.modA.noarch
+           """
 
-  Scenario: I can get info for a disabled module profile when specifying stream
+  Scenario: Get info for an enabled stream, module name and stream specified
+       When I successfully run "dnf module info ModuleA:f26"
+       Then the command stdout should match line by line regexp
+           """
+           ?Last metadata expiration check
+           
+           Name +: +ModuleA
+           Stream +: +f26
+           Version +: +2
+           Profiles +: +client default devel minimal server
+           Repo +: +modularityABDE
+           Summary +: +Module ModuleA summary
+           Description +: +Module ModuleA description
+           Artifacts +: +TestA-1-2.modA.noarch
+            +: +TestB-1-1.modA.noarch
+            +: +TestC-1-2.modA.noarch
+            +: +TestD-1-1.modA.noarch
+           """
+
+  # expected to fail, should be updated when bz1540189 will be resolved
+  Scenario: Get info for an installed profile, module name and profile specified
+       When I successfully run "dnf module info ModuleA/client"
+       Then the command stdout should match regexp "profile specific info or a warning"
+
+  # expected to fail, should be updated when bz1540189 will be resolved
+  Scenario: Get info for an installed profile, module name, stream and profile specified
+       When I successfully run "dnf module info ModuleA:f26/client"
+       Then the command stdout should match regexp "profile specific info or a warning"
+
+  # expected to fail, should be updated when bz1540189 will be resolved
+  Scenario: Get error message when info for non-existent profile is requested
+       When I run "dnf module info ModuleA:f26/non-existent-profile"
+       Then the command should fail
+
+  Scenario: Get error message when info for non-existent module is requested
+       When I run "dnf module info non-existent-module"
+       Then the command should fail
+        And the command stderr should match regexp "Error: No such module"
+
+  Scenario: Get info for a disabled stream, module name and stream specified
        When I successfully run "dnf module info ModuleB:f26"
+       Then the command stdout should match line by line regexp
+           """
+           ?Last metadata expiration check
+           
+           Name +: +ModuleB
+           Stream +: +f26
+           Version +: +2
+           Profiles +: +default
+           Repo +: +modularityABDE
+           Summary +: +Module ModuleB summary
+           Description +: +Module ModuleB description
+           Artifacts +: +TestG-1-2.modB.noarch
+            +: +TestI-1-1.modB.noarch
+           """
+
+  # expected to fail, should pass when bz1540165 will be resolved
+  Scenario: Get info for a disabled stream, when another stream of the same module is locked
```

Comment:
This test should probably be removed in future due to lock/unlock command being removed.

Reply:
Scenario has been removed.